tfexec/terraform_workspace_new.go

```diff
@@ -0,0 +1,19 @@
+package tfexec
+
+import (
+	"context"
+)
+
+//WorkspaceNew creates a new workspace
+func (c *terraformCLI) WorkspaceNew(ctx context.Context, workspace string, dir string, opts ...string) error {
```

Comment:
The `dir` parameter is no longer available in Terraform v0.15.0.
https://github.com/hashicorp/terraform/blob/v0.15/CHANGELOG.md#0150-april-14-2021

The current implementation actually doesn't seem to use it. Can we remove it?

Reply:
removed.